keys/signer_edwards25519_test.go

```diff
@@ -78,7 +78,7 @@ func mockSignature(
 	payload := &types.SigningPayload{
 		AccountIdentifier: &types.AccountIdentifier{Address: "test"},
 		Bytes:             msg,
-		SignatureType:     types.Ed25519,
```

Comment:
Is this an intended change? I am wondering how this hardcoded change worked before?

Reply:
oh I see you refactored at the end.